usaspending_api/references/migrations/0067_create_officers.py

```diff
@@ -0,0 +1,32 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.10.1 on 2017-04-13 17:41
+from __future__ import unicode_literals
+
+from django.db import migrations
+
+
+def update_legal_entities(apps, schema_editor):
```

Comment:
Although _one_ reason to use this pattern is I think it might be faster to do a reverse lookup from this item, rather than search through all LegalEntityOfficers for the specific duns. (i.e. a get_or_create)